docs/src/pages/[platform]/connected-components/liveness/credentials-provider.android.mdx

```diff
@@ -1,28 +0,0 @@
-## Auth
```

Comment:
Can we delete this file if it is unused?

Reply:
Breaks imports I think, that's why the React one is empty as well.  When Swift deletes that file as well we can remove the files and the import